setup.py

```diff
@@ -12,7 +12,8 @@
 import shutil
 
 import torch
-from torch.utils.cpp_extension import BuildExtension, CppExtension, CUDAExtension, CUDA_HOME
+from torch.utils.cpp_extension import BuildExtension, CppExtension, CUDAExtension, CUDA_HOME, ROCM_HOME
```

Comment:
Looks like you'll need to guard the `ROCM_HOME` import somehow, either in torchvision or in PyTorch (I think it would be preferable to do it in PyTorch)

Reply:
Indeed. The easiest probably would be to set `ROCM_HOME` to `None` if it isn't available, just as we do with `CUDA_HOME`. This is in `torch.utils.cpp_extension` itself.

Reply:
I think `ROCM_HOME` value is already set to `None` in pyTorch when it is not built for ROCm. For the CI tests that are running into the error `ImportError: cannot import name 'ROCM_HOME' from 'torch.utils.cpp_extension' `, I noticed that the tests install `torch-1.4.0`. The support of extensions on ROCm went into pyTorch very recently, so it looks like it is trying to read `ROCM_HOME` from pyTorch version it was not put in. 
The CI tests here that are passing, all install `torch-1.5.0.+` and can see the variable.

Reply:
@fmassa @t-vi would you suggest checking for pyTorch version in torchvision for ROCm build?

Reply:
I know the issue came up in previous PRs, but I'm not sure what the conclusion was...

Reply:
I just added a check for torch version around reading `ROCM_HOME`. That should take care of a scenario with older version of pytorch trying to use torchvision with ROCm enabled

Reply:
> I noticed that the tests install torch-1.4.0

This is not quite true I believe, we use PyTorch nightly in our CI, see https://github.com/pytorch/vision/blob/7d1cd1de34298ccb4a993d7c6af59b08cf8ac094/.travis.yml#L37 and https://github.com/pytorch/vision/blob/7d1cd1de34298ccb4a993d7c6af59b08cf8ac094/.circleci/config.yml.in#L42
(or, for simplicity, see the output of one of the CI jobs, like https://circleci.com/gh/pytorch/vision/96101?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link, which shows
```
pytorch:         1.5.0.dev20200305-py3.5_cuda10.0.130_cudnn7.6.3_0 pytorch-nightly
```
)

The change looks ok to me, but I think it can be simplified as we assume that torchvision nightly should use a PyTorch nightly.

Here is what I propose, we can merge this as is and then can you send a follow-up PR cleaning up the checking logic for PyTorch?